Gibe.DittoProcessors/Processors/AbsoluteUrlAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Core.Models;
+using Umbraco.Web;
+
+namespace Gibe.DittoProcessors.Processors
+{
+	public class AbsoluteUrlAttribute : TestableDittoProcessorAttribute
+	{
+
+		public override object ProcessValue()
+		{
+			var content = Value as IPublishedContent;
```

Comment:
Hm, what's the benefit of having two types which perform the same operation here? Any context in which one is valid, the other would never be valid, so it doesn't seem like we gain flexibility.